### PR TITLE
ACMS-1665: Site Studio recommended translation configurations for existing customers

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-1665](https://backlog.acquia.com/browse/ACMS-1665)

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by apathak18 — COMMENTED** [review]
@chandan-singh7929 why accessCheck is FALSE? And can we create only one method for get_latest_revision_id & get_field_layout_canvas_latest_revision_id ~~ we can pass the param for logic implementation.

cc: @vishalkhode1

File: modules/acquia_cms_page/acquia_cms_page.post_update.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        ],
+      ])->save();
+    }
+    $sandbox['current']++;
+  }
+
+  \Drupal::messenger()
+    ->addMessage($sandbox['current'] . ' page processed.');
+
+  if ($sandbox['total'] == 0) {
```
Comment (deepakmishra2) [inline modules/acquia_cms_page/acquia_cms_page.post_update.php:70]: add === to check strict type

**Review by rajeshreeputra — APPROVED** [review]
LGTM

**Comment by chandan-singh7929** [comment]
> @chandan-singh7929 why accessCheck is FALSE? And can we create only one method for get_latest_revision_id & get_field_layout_canvas_latest_revision_id ~~ we can pass the param for logic implementation.
> 
> cc: @vishalkhode1

@ankit we can't use `accessCheck TRUE` its an update hook.
And regarding 
>can we create only one method for get_latest_revision_id & get_field_layout_canvas_latest_revision_id ~~ we can pass the param for logic implementation.

There's group by being used with different fields.

**Comment by chandan-singh7929** [comment]
Closing this one as duplicate of https://github.com/acquia/acquia_cms/pull/1459

**Pull request closed without merging**